pkgs/development/libraries/science/math/osqp/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
```

Comment:
`lib` is actually used only once in `meta = with lib; {`. In such case it would be better to use `meta = with stdenv.lib; {` and remove this line.